### ISSUE TITLE
Add headless Chrome target

### ISSUE THREAD

**Issue by lencioni** [Galooshi/happo#201]
Now that headless Chrome is a thing, it probably shouldn't be too difficult to add Chrome as a rendering target for Happo.

**Comment by trotzig** [comment]
Yep, the only thing different from the firefox target is that the chrome webdriver doesn't support taking a screenshot of a single element (at least not last time I checked). This is the relevant code for the happo-target-firefox package: 
https://github.com/Galooshi/happo/blob/c291f5fa9e729e49c8b1d2246a288da18ed51052/packages/happo-target-firefox/src/runVisualDiffs.js#L76

**Comment by beeing** [comment]
In case if you missed this, puppeteer will be a better option.

An [example of screenshot](https://github.com/GoogleChrome/puppeteer/blob/c866c175f8e7c581ba54fe75c5daf97206697b72/test/test.js#L1888) in their test code.
